src/components/atoms/AvatarDropdown/index.tsx

```diff
@@ -81,7 +96,7 @@ export default function AvatarDropdown() {
                     <Link
                       href={"/"}
                       className="flex items-center p-2 -m-3 transition duration-150 ease-in-out rounded-lg hover:bg-neutral-100 dark:hover:bg-neutral-700 focus:outline-none focus-visible:ring focus-visible:ring-orange-500 focus-visible:ring-opacity-50"
-                      onClick={() => close()}
+                      onClick={() => handleSignOut }
```

Comment:
This is not correct. "Meus Cupons" will handleSignOut? Also `handleSignOut()` please otherwise it won't be called

Reply:
i placed the function in wrong place. thats my bad.